tests/plugins_test.go

```diff
@@ -262,7 +310,7 @@ func getGoModVersion(ctx context.Context, t *testing.T, client *http.Client, mod
 	return cmp.Or(modFile.Go.Version, defaultGoModVersion)
 }
 
-func createBufGenYaml(t *testing.T, basedir string, plugin *plugin.Plugin) error {
+func createBufGenYaml(t *testing.T, basedir string, plugin *plugin.Plugin, goPkgPrefix string) error {
```

Comment:
It feels a bit odd we're adding a go package prefix to a fairly common function.